app/scripts/views/components/pieChart/pieChartTemplate.js

```diff
@@ -40,7 +40,7 @@
               '@mouseenter="mouseEnter($event)" @mouseleave="mouseLeave($event)">' +
               '<chart-operations :has-chart-title="hasChartTitle" :display-name="displayName" :show-table-icon.sync="showTableIcon" ' +
               ' :show-pie-icon.sync="showPieIcon" :chart-id="chartId" :show-operations="showOperations" :groupid="groupid" ' +
-              ':reset-btn-id="resetBtnId" :chart="chartInst" :attributes="attributes"></chart-operations>' +
+              ':reset-btn-id="resetBtnId" :component="piechart" :chart="chartInst" :attributes="attributes"></chart-operations>' +
```

Comment:
'component' is a predefined word in vue. Instead use something like 'component-type'